### ISSUE TITLE
No linting in Vim config

### ISSUE THREAD

**Issue by limpa** [oscekh/.dotfiles#1]
You should add automatic linting to your Vim config. Googles Java standard **demands** two-space indentation, which you seem to have a problem following, and thus using a linter would be appropriate.

**Comment by limpa** [comment]
@oscekh

**Comment by GlennOlsson** [comment]
I agree with @oscekh , I also stand against Big Corpo. Don't let the big man tell you what to do!
